.travis.yml

```diff
@@ -1,15 +1,14 @@
 # This file was generated automatically from conda-smithy. To update this configuration,
-# update the conda-forge.yaml and/or the recipe/meta.yaml.
+# update the conda-forge.yml and/or the recipe/meta.yaml.
 
 language: generic
 
 os: osx
+osx_image: beta-xcode6.1
 
 env:
   matrix:
     
-    - CONDA_PY=27
-    - CONDA_PY=34
     - CONDA_PY=35
```

Comment:
That would be the first thing to check. After all there were some request removals. It is always possible something went wrong.